clients/java/src/test/java/io/camunda/client/impl/http/ApiEntityConsumerTest.java

```diff
@@ -99,7 +106,14 @@ void testJsonApiEntityConsumerWithValidJsonOtherTypeErrorResponse() throws IOExc
     assertThat(entity).isInstanceOf(Error.class);
     final ProblemDetail response = entity.problem();
     assertThat(response).isNotNull();
-    assertThat(response.getTitle()).isEqualTo("Unexpected server response");
+    assertThat(response.getTitle())
+        .isEqualTo(
+            "The entity consumer failed to parse the "
+                + "JSON response: Unrecognized field \"foo\" "
+                + "(class io.camunda.client.protocol.rest.ProblemDetail), not marked as ignorable "
+                + "(5 known properties: \"title\", \"detail\", \"type\", \"instance\", \"status\"])\n"
+                + " at [Source: UNKNOWN; byte offset: #UNKNOWN] (through reference chain: "
+                + "io.camunda.client.protocol.rest.ProblemDetail[\"foo\"])");
     assertThat(response.getDetail()).isEqualTo(jsonResponse);
```

Comment:
🔧 If we adjust according to the suggestion above, we need to adjust the assertions as well.
```suggestion
    assertThat(response.getTitle()).isEqualTo("Cannot parse the server JSON response");
    assertThat(response.getDetail())
        .contains(jsonResponse)
        .contains("The client failed to parse the JSON response")
        .contains("Unrecognized field \"foo\"");
```

Reply:
Updated